packages/compartment-mapper/test/test-main.js

```diff
@@ -83,6 +83,42 @@ with the current test fixture.`);
   assertFixture(t, { namespace, globals });
 });
 
+test('makeBundle / importArchive', async t => {
+  t.plan(fixtureAssertionCount);
+
+  const archiverLocation = new URL(
+    '../src/import-archive.js',
+    import.meta.url,
+  ).toString();
+
+  const archiverBundle = await makeBundle(readPowers.read, archiverLocation);
+  const archiverCompartment = new Compartment({
+    TextEncoder,
+    TextDecoder,
+    URL,
+    assert,
+  });
+  const evasiveArchiverBundle = archiverBundle
+    .replace(/(?<!\.)\bimport\b(?![:"'])/g, 'IMPORT')
+    .replace(/\beval\b/g, 'EVAL');
```

Comment:
can we reuse the transforms or would that setup a dep weird topology

Reply:
The evasive transform (not the censor) is Babel based and in import-bundle, this the layering concern, and why I had to manually evade the compartment.import call.